app/ts/util/PataviUtil.ts

```diff
@@ -24,7 +24,7 @@ import {
   TPerformance
 } from '@shared/interface/Problem/IPerformance';
 import IProblem from '@shared/interface/Problem/IProblem';
-import IPvf from '@shared/interface/Problem/IPvf';
+import IPvf, {TPvf} from '@shared/interface/Problem/IPvf';
```

Comment:
IPvf should not be used

Reply:
could consider not even exporting it and putting the other interface declarations that extend it in the same file instead